staging/src/k8s.io/api/node/v1beta1/types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1beta1
```

Comment:
You also need to add it to the alpha API, I think

Reply:
not positive though... ask sig-apimachinery?

Reply:
@tallclair - I agree, I think we should. We have to be able to convert from internal types to versioned without data loss:

1 - v1beta1 -> internal type works
2 - internal type -> v1alpha1 doesn't work if we don't have `Overhead` field